src/Symfony/Component/String/Inflector/EnglishInflector.php

```diff
@@ -348,11 +348,14 @@ final class EnglishInflector implements InflectorInterface
         // indices (index)
         ['xedni', 5, false, true, ['indicies', 'indexes']],
 
+        // fax (faxes, faxxes)
+        ['xaf', 3, true, true, ['faxes', 'faxxes']],
+
         // boxes (box)
         ['xo', 2, false, true, 'oxes'],
 
         // indexes (index), matrixes (matrix)
-        ['x', 1, true, false, ['cies', 'xes']],
+        ['x', 1, true, false, ['ces', 'xes']],
 
         // appendices (appendix)
         ['xi', 2, false, true, 'ices'],
```

Comment:
Shouldn't this one be moved before `x`?

Reply:
We can actually merge both, I updated the PR